packages/gateway-client/src/worker/index.ts

```diff
@@ -844,12 +884,12 @@ async function main() {
                 async evt => {
                     const connection = evt.detail;
                     const str_id = connection.remotePeer.toString();
-                    console.log(`Connected to ${str_id}, check ${serverID}`);
```

Comment:
FYI, I'm actually adding more logging here not less in my refactor, this info is pretty useful.